addons/addon-base-raas/packages/base-raas-services/lib/study/helpers/entities/study-permissions-methods.js

```diff
@@ -71,6 +71,13 @@ function applyUpdateRequest(studyPermissionsEntity, updateRequest) {
   });
 }
 
+function isValidUpdateRequest(updateRequest) {
+  // Check to make sure the wildcard is not used as a uid in usersToRemove
+  const wildcardEntries = updateRequest.usersToRemove.filter(entry => entry.uid === '*');
```

Comment:
I think using `any` would make this simpler

```suggestion
  return updateRequest.usersToRemove.any(entry => entry.uid === '*');
```

Reply:
I think Javascript uses `some` for this purpose. It did make it simpler! Thanks for pointing out there may be a more efficient array method.